tests/platform_tests/cli/test_show_platform.py

```diff
@@ -130,13 +130,14 @@ def test_show_platform_syseeprom(duthosts, enum_rand_one_per_hwsku_hostname, dut
         for line in syseeprom_output_lines[6:]:
             t1 = regex_int.match(line)
             if t1:
-                parsed_syseeprom[t1.group(2).strip()] = t1.group(4).strip()
+                tlv_code_lower_case = t1.group(2).strip().lower()
+                parsed_syseeprom[tlv_code_lower_case] = t1.group(4).strip()
 
         for field in expected_syseeprom_info_dict:
-            pytest_assert(field in parsed_syseeprom, "Expected field '{}' not present in syseeprom on '{}'".format(field, duthost.hostname))
-            pytest_assert(parsed_syseeprom[field] == expected_syseeprom_info_dict[field],
+            pytest_assert(field.lower() in parsed_syseeprom, "Expected field '{}' not present in syseeprom on '{}'".format(field, duthost.hostname))
+            pytest_assert(parsed_syseeprom[field.lower()] == expected_syseeprom_info_dict[field],
```

Comment:
Can you simply insert a line `field = field.lower()`. Then no need to run `field.lower()` multiple times. And `expected_syseeprom_info_dict[field]` also can benefit from this.

Reply:
I am not sure that the key of expected_syseeprom_info_dict must be lower case.
So maybe we can not set field = field.lower().

Reply:
It looks the expected_syseeprom_info_dict information comes from inventory file. Can you fix the inventory file to match the key?

Reply:
Yes, the expected_syseeprom_info_dict comes form inventory file. But expected_syseeprom_info_dict  is got by 
the following method. I think the current fix will be more robust because it is regardless of the case sensitivity of data in inventory.
```
import sonic_platform.platform as P
info = P.Platform().get_chassis().get_system_eeprom_info()
print(info)
```